util/attach_all.py

```diff
@@ -0,0 +1,86 @@
+import subprocess
+import re
+import sys
+import threading
+import socket
+
+from subprocess import PIPE
+
+outlock = threading.Lock()
+
+
+def parse_ranks():
+    ranks = {}
+    stat = subprocess.run(['mana_status'], stdout=PIPE)
```

Comment:
Let's add logic here to check if mana_status is not in the path or if someone wants to provide an absolute path to the mana_status script.

Reply:
Fixed